src/Compilers/VisualBasic/Portable/Symbols/Source/SynthesizedFieldSymbol.vb

```diff
@@ -59,6 +59,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return
             End If
 
+            Dim compilation = Me.DeclaringCompilation
+            If Type.ContainsTupleNames() AndAlso
+                compilation.HasTupleNamesAttributes AndAlso
```

Comment:
I will keep this code as-is for now. 
 Will look at it again when working on general diagnostics of cases related to TupleNamesAttribute  [https://github.com/dotnet/roslyn/issues/13940](https://github.com/dotnet/roslyn/issues/13940)

---
In reply to: [79723648](https://github.com/dotnet/roslyn/pull/13923#discussion_r79723648) [](ancestors = 79723648)